back/match-service/src/main/java/com/example/match_service/controller/ParticipationController.java

```diff
@@ -0,0 +1,58 @@
+package com.example.match_service.controller;
+
+import com.example.match_service.common.dto.CustomResponseBody;
+import com.example.match_service.common.dto.StatusCode;
+import com.example.match_service.common.resolver.userid.UserId;
+import com.example.match_service.dto.GameResultRequest;
+import com.example.match_service.dto.ParticipationsResponse;
+import com.example.match_service.dto.UserParticipationsResponse;
+import com.example.match_service.service.ParticipationService;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ParticipationController {
+
+    private ParticipationService participationService;
+
+    @PostMapping("/participations/{matchId}")
+    public ResponseEntity<CustomResponseBody<Void>> applyMatch(@UserId UUID userId, @PathVariable Long matchId) {
+        //TODO userservice로부터 userTagResponse 받기
+        participationService.applyToMatch(matchId,userId);
+        return ResponseEntity.ok(new CustomResponseBody<>(StatusCode.SUCCESS, "참가 신청되었습니다."));
+    }
+
+    //참여자 조회
+    @GetMapping("/participations/matches/{matchId}")
+    public ResponseEntity<CustomResponseBody<List<ParticipationsResponse>>> getParticipations(
+            @UserId UUID userId,
+            @PathVariable Long matchId) {
+
+        return ResponseEntity.ok(new CustomResponseBody<>(
+                StatusCode.SUCCESS,
+                "요청이 성공적으로 처리되었습니다.",
+                participationService.getParticipations(userId, matchId)));
```

Comment:
```suggestion
        return ResponseUtil.success(participationService.getParticipations(userId, matchId), "요청이 성공적으로 처리되었습니다.");
```

이런 식으로 `ResponseUtil` 쓰면 더 깔끔해질 것이어요~~ 필수는 아닙니다.. ㅎㅎ

Reply:
헉 제가 잘 활용을 못하고 있었네유 감사합니다! 훨씬 깔끔하군여,,, 반영해두겠슴다